src/main/scala/gitbucket/core/model/activity/tag/TagActivityInfo.scala

```diff
@@ -0,0 +1,53 @@
+package gitbucket.core.model.activity.tag
```

Comment:
I feel a little uncomfortable in the package structure of ActivityInfo classes. For example, `TagActivityInfo` and `BranchActivityInfo` should be categorized as `git` separated from `IssueActivityInfo`, etc. Also, `ForkActivityAction` should be categorized as `repository`.

I think simply putting all ActivityInfo classes under `activity` package would be fine because in other types of components like controllers or models also have a flat package structure.

Reply:
That sounds good to me, I can put all the classes under the `activity` package, since this project appears to favour a flat package structure.